ui/src/data-services/hooks/taxalists/useTaxalists.ts

```diff
@@ -0,0 +1,49 @@
+import { API_ROUTES } from 'data-services/constants'
+import { ServerTaxaList, TaxaList } from 'data-services/models/taxalist'
+import { FetchParams } from 'data-services/types'
+import { getFetchUrl } from 'data-services/utils'
+import { useMemo } from 'react'
+import { UserPermission } from 'utils/user/types'
+import { useAuthorizedQuery } from '../auth/useAuthorizedQuery'
+
+// Convert server response to a TaxaList instance
+const convertServerRecord = (record: ServerTaxaList) => new TaxaList(record)
+
+export const useTaxaLists = (
```

Comment:
I think this whole file can be removed, seems to be a repetition of `ui/src/data-services/hooks/taxa-lists/useTaxaLists.ts`.